frontend/public/components/app.jsx

```diff
@@ -130,6 +130,9 @@ class App extends React.PureComponent {
         <Switch>
           <Route path={['/all-namespaces', '/ns/:ns',]} component={RedirectComponent} />
 
+          <LazyRoute path="/overview/ns/:ns" exact loader={() => import('./overview' /* webpackChunkName: "overview" */).then(m => m.OverviewPage)} />
```

Comment:
Is there not an overview for all namespaces?

Reply:
> Is there not an overview for all namespaces?

I'm worried how this will scale in large clusters. We might need an empty state "select a project" message in the short term until we can figure it out.